src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -78,20 +79,25 @@ public async override Task RunAsync()
             if (extensionBundleManager.IsExtensionBundleConfigured())
             {
                 var hostFilePath = Path.Combine(Environment.CurrentDirectory, ScriptConstants.HostMetadataFileName);
-                if (_showExtensionBundleWarning)
+                if (_showNoActionWarning)
                 {
-                    ColoredConsole.WriteLine(WarningColor($"No action performed. Extension bundle is configured in {hostFilePath}"));
+                    ColoredConsole.WriteLine(WarningColor($"No action performed. Extension bundle is configured in {hostFilePath}."));
                 }
                 return;
             }
 
-            if (CommandChecker.CommandExists("dotnet"))
+            if (!string.IsNullOrEmpty(ConfigPath) && !FileSystemHelpers.DirectoryExists(ConfigPath))
             {
-                if (!string.IsNullOrEmpty(ConfigPath) && !FileSystemHelpers.DirectoryExists(ConfigPath))
-                {
-                    throw new CliArgumentsException("Invalid config path, please verify directory exists");
-                }
+                throw new CliArgumentsException("Invalid config path, please verify directory exists");
+            }
+
+            if (!NeedsExtensionsInstall())
```

Comment:
`!string.IsNullOrEmpty(Package) && !string.IsNullOrEmpty(Version)` is being ignored here. We need to not skip in extension installation incase user has specified individual extension id and version.